_maps/shuttles/whiteship_delta.dmm

```diff
@@ -0,0 +1,213 @@
+"aa" = (/turf/open/space/basic,/area/space)
```

Comment:
TGM.

Reply:
I did use TGM, though I fix that within the hour when I get the chance.